ods_ci/utils/scripts/ocm/ocm.py

```diff
@@ -502,15 +502,6 @@ def hide_values_in_op_json(self, fields, json_str):
                     p["value"] = "##hidden##"
         return json.dumps(json_dict)
 
-    def hide_values_in_op_json(self, fields, json_str):
-        json_dict = json.loads(json_str)
-        params = json_dict["parameters"]["items"]
-        for field in fields:
-            for p in params:
-                if p["id"] == field:
-                    p["value"] = "##hidden##"
-        return json.dumps(json_dict)
-
```

Comment:
why deleting this function? it is used in `install_addon` function

Reply:
Ah it's duplicated, ok

Reply:
because it is duplicated on lines 496 to 503.